ui/src/components/ToS.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+import {LitElement, html, css} from 'lit-element';
+
+import {ToSService} from '../utils/ToSService';
+
+import {ListItem} from '@material/mwc-list/mwc-list-item';
+ 
+/**
+ * Component responsible for providing users a means to agree
+ * to the terms of service. 
+ */
+export class ToS extends LitElement {
+
+    static get properties() {
+        return {
+            country: {type: String},
+            tos: {type: Array}
+        };
+    }
+
+    static get styles() {
+        return css`
+            textarea.terms {
+                border: none;
+                font-family: 'Roboto';
+                font-size: 15px; 
+                height: 85vh;
+                width: 90%;
+                overflow: scroll; 
+                padding: 5px 20px 0px 20px; 
+            }
+            div.button-container {
+                align-items: center; 
+                box-shadow: 0px -3px 1px -1px #dcdcdc;
+                display: flex; 
+                flex-direction: row; 
+                flex-wrap: wrap;
+                height: 12vh;
+                justify-content: center; 
+                text-align: center;  
+            }
+            mwc-button {
+                font-size: 8x;
+                padding: 10px;
+            }
+            mwc-button.accept {
+                --mdc-theme-primary: #1a73e8;
+                --mdc-theme-on-primary: white;
+                width: 60vw;
+            }
+            mwc-button.cancel {
+                --mdc-theme-primary: white;
+                --mdc-theme-on-primary: #1a73e8;
+                width: 15vw;
+            }
+        `;
+    }
```

Comment:
move to its own style file
edit: it has its own style file in the other PR? are you missing a commit here from the other branch? might wanna merge.. also a bunch of other files here are added for the "first" time despite being in the other PR as well

Reply:
Totally my fault--the other pr is the most up-to-date, files got accidentally merged